cli/step_info.go

```diff
@@ -9,20 +9,20 @@ import (
 )
 
 func printStepLibStep(collectionURI, id, version, format string) error {
+	stepInfo, err := tools.StepmanStepInfo(collectionURI, id, version)
+
 	switch format {
 	case output.FormatRaw:
-		out, err := tools.StepmanRawStepLibStepInfo(collectionURI, id, version)
-		if out != "" {
-			fmt.Println("Step info:")
-			fmt.Printf("%s", out)
+		if err != nil {
+			return err
 		}
-		return err
+		fmt.Println("Step info:")
+		fmt.Println(stepInfo.String())
 	case output.FormatJSON:
-		outStr, err := tools.StepmanJSONStepLibStepInfo(collectionURI, id, version)
 		if err != nil {
 			return fmt.Errorf("StepmanJSONStepLibStepInfo failed, err: %s", err)
 		}
-		fmt.Println(outStr)
+		fmt.Println(stepInfo.JSON())
 		break
```

Comment:
just a minor nitpicking comment: there are two unnecessary `break` statements which were already there but could be removed.

Reply:
Updated in https://github.com/bitrise-io/bitrise/pull/820/commits/2bfaba903eb6ea234c8604f69cb6f91eb740e26c